share/wake/lib/system/io.wake

```diff
@@ -36,6 +36,7 @@ target writeImp inputs mode path content =
                     Pass path = Pass (editRunnerOutputOutputs (path, _) output)
         makeRunner "write" (\_ Pass 0.0) pre post virtualRunner
     makeExecPlan ("<write>", str mode, path, Nil) inputs
+    | setPlanLabel "<write> {str mode} {path}"
```

Comment:
I was going to say something about Plans without labels potentially falling back to the command line, but I realized that even if that's what we *show*, that's not what's *stored*.  This line may or may not change the live output, but it is definitely important for making the database queries look nice.  (`wake --last --metadata | grep Job` is only of limited use without it.)